src/condor_startd.V6/ResAttributes.cpp

```diff
@@ -2010,6 +2013,8 @@ CpuAttributes::publish_dynamic(ClassAd* cp) const
 		cp->Assign( ATTR_LOAD_AVG, rint((c_owner_load + c_condor_load) * 100) / 100.0 );
 		cp->Assign( ATTR_KEYBOARD_IDLE, c_idle );
 
+		// dprintf(D_IDLE, "KeyboardIdle=%lld LoadAvg=%.2f\n", c_idle, rint((c_owner_load + c_condor_load) * 100) / 100.0);
+
```

Comment:
Do we want to keep this `dprintf`?

Reply:
not sure prefer to leave it commented out for now.